recipes/robotraconteur/bld.bat

```diff
@@ -0,0 +1,19 @@
+@echo off
+
+md build2
+cd build2
+if %errorlevel% neq 0 exit /b %errorlevel%
+cmake -GNinja -DBUILD_TESTING=OFF -DBUILD_DOCUMENTATION=OFF -DBUILD_GEN=ON -DBUILD_PYTHON3=ON ^
+  -DCMAKE_BUILD_TYPE:STRING=Release -DBUILD_SHARED_LIBS:BOOL=ON ^
+  -DPYTHON3_EXECUTABLE=%PYTHON% ^
```

Comment:
This should probably be `Python3_EXECUTABLE` (capitalisation is important; also for build.sh)

Reply:
I have my own Python routines that use this capitalization.